lib/launchers/base.js

```diff
@@ -29,7 +29,17 @@ function BaseLauncher (id, emitter) {
   this.emitAsync = KarmaEventEmitter.prototype.emitAsync.bind(this)
 
   this.id = id
-  this.state = null
+  this._state = null
+  Object.defineProperty(this, 'state', {
+    get: function () {
+      return this._state
+    },
+    set: function (toState) {
+      this.log.debug(`${this._state} -> ${toState}`)
```

Comment:
should be just `log` instead of `this.log`?

Reply:
yes thanks